lib/Whim/Core.pm

```diff
@@ -202,15 +201,10 @@ sub process_webmentions( $self ) {
         # Grab the author image
         my $photo_hash;
```

Comment:
Does this still need to be here? It looks like it's only used within the if block below.